Add responsive size

Tatayoyoh opened the pull request:
Hi @dbetka !
I would like to contribute to your great package by adding responsive size like '2.5em' or '250%'.

Value can be passed as app project configuration
```
import materialSymbolsPlugin from '@dbetka/vue-material-symbols';
import 'material-symbols/index.css';

const app = createApp(App);
app.use(materialSymbolsPlugin, {
  defaultSize: '1.5em',
});
```

or attribute
```
<MaterialSymbol name="delete" size="1.5em">
<MaterialSymbol name="delete" size="150%">
```

Can you validate this evolution and release a new package version if ok ?
Thanks a lot for your code :)

Tatayoyoh committed:
Add responsive size

dbetka commented on src/MaterialSymbol.vue:
```diff
@@ -31,11 +33,12 @@ const styles = computed(() => {
   const weight = props.weight
   const grade = GRADES[props.grade]
-  const size = props.size
+  const size = Number.isIntefer(props.size) ? `${props.size}px` : props.size
+  const opsz = Number.isIntefer(props.size) ? props.size : window.$materialSymbolsDefaults?.defaultSize || DEFAULT_SIZE,
```
```suggestion
  const size = Number.isInteger(props.size) ? `${props.size}px` : props.size
  const opsz = Number.isInteger(props.size) ? props.size : window.$materialSymbolsDefaults?.defaultSize || DEFAULT_SIZE
```

dbetka reviewed (changes requested):
@Tatayoyoh Idea looks good and it seems to work after small corrections.
I think we should add some relative value to `demo/DemoComponent.vue`.

For example change this
```js
const sizes = [ undefined, 40, 56, 72, 96, 128 ]
```
to this
```js
const sizes = [ undefined, 40, 56, 72, 96, '7.5rem' ]
```

Are you able to make this modifications? 
After that I will release a new package version.

Tatayoyoh committed:
Fix code errors and add update DemoComponent.vue example

Tatayoyoh commented:
Hey, I fixed two ugly code errors from my previous pull request and added the example into `demo/DemoComponent.vue` like you said. I can see conflicts now, sorry I have not seen you made changes :open_mouth:

Tatayoyoh committed:
Merge branch 'master' into master

dbetka commented on demo/DemoComponent.vue:
```diff
@@ -1,10 +1,8 @@
 
-const { metadata } = useSymbols()
-console.log(metadata)
+const sizes = [ undefined, 40, 56, 72, 96, "7.5em" ]
```
Single quote is preferable by eslint but I will change it right before release.

dbetka reviewed (approved):
PR is ready to merge and release today - Thank you for your commitment :)

dbetka commented:
@Tatayoyoh new version has been released 🚀 
Check if everything works fine

Tatayoyoh commented:
Thank you a lot for this ! :1st_place_medal:
